db/migrate/20161003145646_add_position_to_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToLinks < ActiveRecord::Migration
+  def change
+    add_column :links, :position, :integer
```

Comment:
If we were to set this as non-null with a default of 0 could we drop the other migration?